src/libs/actions/Report.ts

```diff
@@ -1160,8 +1160,11 @@ function expandURLPreview(reportID: string, reportActionID: string) {
     API.read(READ_COMMANDS.EXPAND_URL_PREVIEW, parameters);
 }
 
-/** Marks the new report actions as read */
-function readNewestAction(reportID: string) {
+/** Marks the new report actions as read
+ *
+ * @param shouldResetUnreadMarker Should we reset unread indicator
```

Comment:
Thanks for adding! but this comment is self-explanatory by reading the name. Can you add more details like in which particular case we're adding it to not reset the marker